### ISSUE TITLE
Not really an issue

### ISSUE THREAD

**Issue by emitanaka** [garthtarr/sydney_xaringan#3]
I think this is really nice and that you should submit the theme to `xaringan`! 
Your scroll feature of the code is particularly noteworthy.

**Comment by garthtarr** [comment]
Yep, good idea. Need to tidy it up a bit and remember where I borrowed some of the code from (for appropriate attribution). Also the logo and images can't really be distributed.
